tests/integration/test_stages.py

```diff
@@ -731,7 +730,9 @@ def test_ta_grading(self):
         self.project_api_mock.get_user_preferences = mock.Mock(
             return_value={UserAwareXBlockMixin.TA_REVIEW_KEY: group_id}
         )
-        self.project_api_mock.get_user_roles_for_course = mock.Mock(return_value=[{'role': ALLOWED_OUTSIDER_ROLES[0]}])
+        self.project_api_mock.get_user_roles_for_course = mock.Mock(return_value=[
+            {'role': AuthXBlockMixin._DEFAULT_TA_ROLE[0]}  # pylint:disable=protected-access
```

Comment:
@jbzdak `_DEFAULT_TA_ROLE` -> `DEFAULT_TA_ROLE` instead of suppression. I don't think we need to make constants private.